acquisitions/src/main/resources/thunderjet/mod-orders/features/routing-lists-api.feature

```diff
@@ -0,0 +1,118 @@
+@parallel=false
+Feature: Test routing list API
+
+  Background:
+    * url baseUrl
+    * callonce loginAdmin testAdmin
+    * def okapitokenAdmin = okapitoken
+    * callonce loginRegularUser testUser
+    * def okapitokenUser = okapitoken
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json'  }
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': '*/*'  }
+
+    * callonce variables
+
+    * def fundId = callonce uuid1
+    * def budgetId = callonce uuid2
+    * def orderId = callonce uuid3
+    * def poLineId = callonce uuid4
+
+    * def rListId1 = callonce uuid5
+    * def rListId2 = callonce uuid6
+    * def rListUserId = callonce uuid7
+
+    * def createOrder = read('../reusable/create-order.feature')
+    * def createOrderLine = read('../reusable/create-order-line.feature')
+
+    * configure headers = headersAdmin
```

Comment:
can we use headersUser here?
related comment: https://github.com/folio-org/folio-integration-tests/pull/1309#issuecomment-2032434390